build/setup.d/21-install-instana-agent.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+#check if INSTANA_AGENT_KEY exists, if not cancel the installation process.
+if [-z "${INSTANA_AGENT_KEY}"]; then
```

Comment:
Do we want `INSTANA_AGENT_KEY` to be mandatory or optional? If it should be optional, then maybe just log the error and proceed without exit?